### DISCUSSION TITLE
翻译简化了，想要详细优点

### DISCUSSION THREAD

**Discussion by wysjwxm** [YiiGuxing/TranslationPlugin#5617] [Q&A]
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/151387734/79638451-0c2d-49bf-8c80-62733c1dea7b)
如题，想要翻译显示多个释义。前端时间翻译结果突然就都简化了，已经不能满足日常使用需求了（哭）

**Comment by YiiGuxing** [comment]
你是说有道翻译的吧？有道原本是有一些额外的信息的，但现在已经被取消掉了，不再返回，原因不明，后面是否会恢复也不明。

**Reply by wysjwxm** [reply]
确实，我换了其他翻译引擎，都没有哪种详细的信息，很苦恼
